defaults/main.yml

```diff
@@ -71,6 +81,14 @@ squid_http_access:
     acl:
       - 'CONNECT'
       - '!SSL_ports'
+  - action: 'allow'
+    acl:
+      - all
+      - domain_whitelist
+  - action: 'deny'
+    acl:
+      - all
+      - domain_blacklist
```

Comment:
This configuration looks like nonsense.

Reply:
I'm afraid you'll need to be more specific about what "nonsense" means. 
Keeping in line with squid.conf template, it results in `http_access deny all domain_blacklist` line which is what we want. 
I can explicitly add this line before http_access block , but then it will always be explicitly 1st on the list (which I'm not sure if it will require reordering at this point). 
Ditto for requested whitelists, not even sure what's the point of them at this point since every domain is allowed.